docker/docker-compose.yml

```diff
@@ -15,9 +15,9 @@ services:
       - PULSE_SERVER=tcp:host.docker.internal:4713
     restart: unless-stopped
     volumes:
-      - ../shared/audiofolders:/home/pi/RPi-Jukebox-RFID/shared/audiofolders
-      - ../shared/playlists:/home/pi/.config/mpd/playlists
-      - ./config/docker.pulse.mpd.conf://home/pi/.config/mpd/mpd.conf
+      - ../shared/audiofolders:/root/RPi-Jukebox-RFID/shared/audiofolders
```

Comment:
Sry for getting at it again, but i still think this should be ${HOME} as it is supplied as args and defaults to "root".
So right now if the user will provide different values it breaks.